siobrultech_protocols/gem/protocol.py

```diff
@@ -167,7 +167,7 @@ class ProtocolStateException(Exception):
 class BidirectionalProtocol(PacketProtocol):
     """Protocol implementation for bi-directional communication with a GreenEye Monitor."""
 
-    def __init__(self, queue: asyncio.Queue[Packet]):
+    def __init__(self, queue: asyncio.Queue):
```

Comment:
This is causing a lint error.  The `from __future__ import annotations` should make this okay for python 3.8 (that's what I had to add to get CI passing)